client/state/stats/lists/actions.js

```diff
@@ -47,23 +61,29 @@ export function requestSiteStats( siteId, statType, query ) {
 			siteId,
 			query,
 		} );
-		const isUndocumented = includes(
-			[
-				'statsFileDownloads',
-				'statsAds',
-				'statsOrders',
-				'statsTopSellers',
-				'statsTopCategories',
-				'statsTopCoupons',
-				'statsTopEarners',
-				'statsStoreReferrers',
-			],
-			statType
-		);
+
+		let subpath;
+		let apiNamespace;
+		if ( wpcomV1Endpoints.hasOwnProperty( statType ) ) {
+			subpath = wpcomV1Endpoints[ statType ];
+			apiNamespace = 'rest/v1.1';
```

Comment:
You could also leave `apiNamespace` undefined here.

Reply:
Yeah, I just decided to keep it for clarity and a better understanding of what's going on.

Reply:
But you were right of course! This broke something - see https://github.com/Automattic/wp-calypso/pull/57814 and https://github.com/Automattic/wp-calypso/issues/57775